src/components/buttons/button/Button.tsx

```diff
@@ -97,6 +97,7 @@ export const MbButton = ({
         className={`button ${btnType} ${state} ${
           customClass ? customClass : `${size} ${getFontType(size)}`
         } relative`}
+        disabled={!!(state === EState.DISABLED)}
```

Comment:
thank you so much for your contribute @marceloF5

just a quick question, why are you forcing the !! when state has the same type?

Reply:
same! https://github.com/Mintbase/mintbase-ui/pull/383#discussion_r982632810